tests/unit/clients/python/test_client.py

```diff
@@ -60,12 +61,14 @@ def test_check_input_fail(input_fn):
 def test_gateway_ready(port_expose, route, status_code):
     p = set_gateway_parser().parse_args(['--port-expose', str(port_expose)])
     with RESTGatewayPea(p):
+        time.sleep(0.5)
```

Comment:
Why are we having a sleep?

Reply:
`uvicorn` takes a small amount of time to get started. This is to allow the threads to get started.

Reply:
Ah ok, got it!